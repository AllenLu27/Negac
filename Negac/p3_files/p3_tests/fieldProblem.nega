struct base {
 int a;
};

void main() {
 struct base a;
 if(a.a != 5) {
 a = false;
 }
}

