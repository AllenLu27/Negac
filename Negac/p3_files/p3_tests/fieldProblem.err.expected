FATAL [8,4]: Invalid assignment operation
Semantic analysis failed
